src/xcode/ENA/ENA/Resources/Localization/ru.lproj/Localizable.strings

```diff
@@ -0,0 +1,888 @@
+
+"OnboardingInfo_howDoesDataExchangeWorkPage_imageDescription" = "Зашифрованная информация о положительном тесте передается в систему, и другим пользователям отправляются предупреждения.";
+
+"OnboardingInfo_howDoesDataExchangeWorkPage_title" = "Если у Вас положительный тест на COVID-19...";
```

Comment:
```suggestion
"OnboardingInfo_howDoesDataExchangeWorkPage_title" = "Если у Вас положительный тест на COVID-19…";
```

Use of … sign instead of 3 dots? (see also line 206)

Reply:
our translation tool can't handle unicode signs